### DISCUSSION TITLE
create our own vulnerability scan jobs

### DISCUSSION THREAD

**Discussion by RasheedJad** [aquasecurity/trivy-operator#923] [Development]
Hi there,

I would like to know if there is the possibility that I can create a scanning job (i.e scan-vulnerabilityreport-*) for a set images and then the operator does the rest of work like creating the custom report and also the Prometheus metrics. is that possible?

**Comment by chen-keinan** [comment]
@RasheedJad the answer is yes, if you will create the scan job with the `appropriate labels` on trivy-operator `namespace`,  the operator will reconcile the scan job and create reports and metrics

**Reply by RasheedJad** [reply]
@chen-keinan any idea about the "appropriate labels" or where are they mentioned in docs or somewhere else so I use them?

**Reply by chen-keinan** [reply]
no docs for it ,you can check the code , however the simple way is run a scan and view it job description.
like this :  `kubectl get <scan-job> -n trivy-system -o yaml`
